src/exchange/CeloNewsFeedItem.tsx

```diff
@@ -18,7 +20,9 @@ interface Props {
 
 export default function CeloNewsFeedItem({ article, testID }: Props) {
   function onPress() {
-    navigate(Screens.WebViewScreen, { uri: article.link })
+    const url = article.link
+    ValoraAnalytics.track(CeloNewsEvents.celo_news_bottom_read_more_tap, { url })
```

Comment:
this should be `celo_news_article_tap` right?

Reply:
yes 🤦